src/Response/Response.ts

```diff
@@ -0,0 +1,56 @@
+/// <reference path="../.d.ts"/>
+"use strict";
+
+import Body from "../Body";
+import { ResponseHeaders } from "../Headers";
+import { isValidRedirectStatus } from "../utils/http";
+
+export default class Response extends Body implements FetchResponse {
+  type: string;
+  url: string;
+  status: number;
+  ok: boolean;
+  statusText: string;
+  headers: ResponseHeaders;
+
+  constructor(body, init: FetchResponseInit = {}) {
```

Comment:
`body = ""`

Then you shouldn't have to supply any args in the `fetch` method.